data/channels.csv

```diff
@@ -25690,7 +25690,7 @@ WBSTV.kr,WBS TV,원음방송,,,KR,KR-11,,c/KR;s/KR-11,kor,religious,FALSE,,,,htt
 WBSTV.ug,WBS TV,,,,UG,,,c/UG,eng,,FALSE,,,,http://wbs.ug/,https://i.imgur.com/ynKBHPh.png
 WBTDLD2.us,WBTD-LD2,,Bounce,,US,US-VA,Suffolk,s/US-VA,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/thumb/e/e0/Bounce_TV_logo.svg/512px-Bounce_TV_logo.svg.png
 WBTDLD3.us,WBTD-LD3,,H&I,,US,US-VA,Suffolk,s/US-VA,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/thumb/9/9e/Heroes_%26_Icons_logo.svg/512px-Heroes_%26_Icons_logo.svg.png
-WBTSCD1.us,WBTS-CD1,,NBC,,US,US-NH,Nashua,s/US-NH,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/9/9f/NBC_Peacock_1986.svg/512px-NBC_Peacock_1986.svg.png
+WBTSCD1.us,WBTS-CD1,,NBC,,US,US-NH,Nashua,s/US-NH,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/c/c6/WBTS-LD_NBC_10_Boston_logo.png

```

Comment:
Image size should not be more than 512x512 px.

https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels

Reply:
fixed in latest update